index.html

```diff
@@ -217,5 +217,53 @@ <h2 class="skill-name">Skills</h2>
 			</li>
 		</ul>
 	</section>
+	<section  class="contact-container" id="footer-container-id">
+		<div class="contact-form">
+			<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>		
+			<form action="https://formspree.io/f/xknagbpo" method="post">
+				<ul class="contact-elements">
+					<li class="text-input">
+						<input type="text" maxlength="30" placeholder="Full name" required>
+					</li>
+					<li>
+						<input type="email" placeholder="Email Address" required>
+					</li>
+					<li>
+						<textarea  type="text" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10"></textarea>
+					</li>
+					<li>
+						<button type="submit">Get in touch</button>
+					</li>
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed) should be implemented as required. Kindly reference the Figma design and fix that.

![image](https://user-images.githubusercontent.com/29071984/213037053-c3e0440a-7125-4f29-8444-a94e3ebef385.png)